packages/cojson/src/coValues/coStream.ts

```diff
@@ -306,20 +320,9 @@ export class RawBinaryCoStreamView<
             const chunk = base64URLtoBytes(
                 item.chunk.slice(binary_U_prefixLength),
             );
```

Comment:
**observation:** I can see this sync decoding become a problem with large files

How do you feel about making `getBinaryChunks` async and see how a worklet would perform here? @aeplay